blueprints/ember-electron/index.js

```diff
@@ -84,8 +84,8 @@ module.exports = class EmberElectronBlueprint extends Blueprint {
       // add install commands -- install dependencies in electron-app project,
       // and export display and launch xvfb
       doc.install = doc.install || [];
-      let entry = doc.install.find(entry => entry.includes('yarn ') || entry.includes('npm '));
-      if (entry.includes('yarn')) {
+      let entry = doc.install.find(entry => entry.includes('yarn '));
+      if (entry) {
```

Comment:
Am I correct in understanding this to mean that we assume use of `npm` if we don't see `yarn`? (I think that's fine though could get pedantic and talk about [`bolt`](https://github.com/boltpkg/bolt) or something)

Reply:
Looks like this was fixed in the latest commits (with `hasInstallSection`)